biome.json

```diff
@@ -4,6 +4,8 @@
         "include": [
             "*.json",
             "*.md",
+            "bin/*.js",
+            "test/*.js",
```

Comment:
Jumping on the occasion, could you please change line 18 ? 👼 

```diff
-            "ignore": ["**/composer.json", "**/vendor", "**/package.json", "**/node_modules"] 
+            "ignore": ["**/composer.json", "**/vendor", "**/package.json", "**/node_modules", "ux.symfony.com/**"]
```

Reply:
Why?

Reply:
Ho and `**/var/` too :)

Reply:
I don't think this is necessary, since we configured Biome `files.includes`

Reply:
It does when i run check-format from the repo root

<img width="1444" alt="image" src="https://github.com/user-attachments/assets/c9174286-0df3-43a0-a88b-5570a76f7c37">

Reply:
Ah, didn't know you had `.json` files under `ux.symfony.com/.....`. Is that a SF 7.2 thing?

Reply:
Yep it is, fixed